db/migrate/20190821095338_change_time_due_to_datetime.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeTimeDueToDatetime < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :moves, :time_due
```

Comment:
I don't think we can get away with this now that we are in production. We have to rename the old column, create the new one, copy the data from old to new and then drop the old column.

Reply:
All the moves in production have a time_due that is set to null, so dropping and recreating the column in the database is safe

Reply:
fair enough